samples/msal-node-auth-code/index.js

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Is this sample runnable? Add a quick readme to run it?

Reply:
Yep, will add a README with instructions on how to run before we release.